benchmarks/reframe_config.py

```diff
@@ -768,6 +768,7 @@ def spack_root_to_path():
                         'reframe %(version)s|'
                         '%(check_info)s|'
                         '%(check_jobid)s|'
+                        '%(check_job_nodelist)s|'
```

Comment:
This was actually already added in #291 so this PR would add a duplicate of it. Therefore I'd ask you to remove it from here.

```suggestion

```

Reply:
This might also be why the CI is failing, because the columns in the perflog are not unique